book/src/concepts/state.md

```diff
@@ -183,6 +183,7 @@ If you were to replace `()` by `MyEventType`, then you could react to them from
 # use amethyst::prelude::*;
 # use amethyst::renderer::VirtualKeyCode;
 # use amethyst::input::is_key_down;
+# struct GameplayState;
 
```

Comment:
https://github.com/amethyst/amethyst/pull/962/files#diff-9c67de6309fc1e25b360604449cc8301R193

what is MyState used for here?

Reply:
I don't know, git blame you.